source/article_draft.html

```diff
@@ -1 +1,103 @@
-<h1>記事ページドラフト</h1>
\ No newline at end of file
+<!DOCTYPE html>
+<html lang="en-US">
+
+	<head>
+		<meta charset="UTF-8" />
+		<title>Tatami pakuri kiji</title>
+	</head>
+
+	<body>
+		<header id="body-header">
+			<hgroup class="body-header-title">
+				<h1 class="title"><a href="http://themes.elmastudio.de/tatami/" title="Tatami">Tatami</a></h1>
+				<h2 class="description">H2のサンプルテキスト</h2>
+			</hgroup>
+		</header>
+		
+		<div class="container">
+		
+			<section class="sidebar-left"><!-- 左側のサイドバーについて記述　-->
```

Comment:
`<div class="sidebar-column is-left"></div>`

`<div class="sidebar-column is-right"></div>`

```
.sidebar-column {
  // 共通の処理
}

.sidebar-column.is-left {
  // 左側だけの処理
}

.sidebar-column.is-right {
  // 右側だけの処理
}

```